inc/saiacl.h

```diff
@@ -277,6 +277,15 @@ typedef enum _sai_acl_action_type_t
     /** Set Forwarding class */
     SAI_ACL_ACTION_TYPE_SET_FORWARDING_CLASS = 0x00000034,
 
+    /** Set ARS monitoring */
+    SAI_ACL_ACTION_TYPE_SET_ARS_MONITORING = 0x00000035,
+
+    /** Enable ARS forwarding */
+    SAI_ACL_ACTION_TYPE_ENABLE_ARS_FORWARDING = 0x00000036,
```

Comment:
can we also consider the AR activation based on router lookup (e.g. Route will have the reference to ARS object) ? Today it is done only based on ACL match. For LAG the FDB lookup can be considered for AR enabling/triggering

Reply:
These are override controls on a per flow basis.
Route lookup will result in ARS enabled next hop group.

Reply:
in some cases it would be beneficial to use different ARS objects (kind of profiles) for different types of traffic. For example, Flowlet for TCP (to minimize out-of-order) and Random_Packet for UDP. Please see my comment on sainexthopgroup.h

Reply:
SAI_ACL_ACTION_TYPE_ENABLE_ARS_FORWARDING is a dynamic action in the switch pipeline doesn't make sense as enabling ARS on a nexthgroup needs ARS related attributes to be set on nexthopgroup object by the control plane.

I will removed this attrbute. Onlu ACL action possible is disabling ARS when a packet is going through ARS enabled nexthopgroup object.